src/Compilers/CSharp/Test/Semantic/Semantics/SlowTests.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests.Semantics
+{
+    public class SlowTests : CSharpTestBase
+    {
+        [Fact, WorkItem(35949, "https://github.com/dotnet/roslyn/issues/35949")]
+        public void NotNull_Complexity()
```

Comment:
>NotNull_Complexity [](start = 20, length = 18)

:memo: this test doubled the time to run nullable tests, so I'm moving it out.

Reply:
This test is not actually related to the PR, we are just moving it for developer quality-of-life when running all nullable tests.